game/ui/general/savemenu.cpp

```diff
@@ -304,23 +304,24 @@ void SaveMenu::tryToSaveGame(const UString &saveName, const sp<Control> parent)
 		// If the user use the first row but wrote the name of an already existing save game
 		else
 		{
-			SaveMenu::askUserIfWantToOverrideSavedGame(saveGameMetadata);
+			SaveMenu::askUserIfWantToOverrideSavedGame(saveGameMetadata, saveName);
 		}
 	}
 
 	// If saving item using row for existing item
 	else
 	{
-		auto slot = parent->getData<SaveMetadata>();
-		SaveMenu::askUserIfWantToOverrideSavedGame(slot);
+		const auto slot = parent->getData<SaveMetadata>();
+		SaveMenu::askUserIfWantToOverrideSavedGame(slot, saveName);
 	}
 }
 
-void SaveMenu::askUserIfWantToOverrideSavedGame(const sp<SaveMetadata> saveMetadata)
+void SaveMenu::askUserIfWantToOverrideSavedGame(const sp<SaveMetadata> saveMetadata,
+                                                const UString &saveName)
 {
-	const auto &saveName = saveMetadata->getName();
+	const auto &metadataSaveName = saveMetadata->getName();
```

Comment:
While we're here, this sort of string should be wrapped in tr() to allow translate.

Probably best to have a separate PR, but don't want to miss stuff like this :)

Reply:
Added `tr()` call to both message box title and content:

```c++
	const auto messageBoxTitle = tr("Override saved game");
	const auto messageBoxContent = tr("Do you really want to override " + metadataSaveName + "?");
```

Reply:
tr() uses the passed in string as a key - so using "metadataSaveName" as part of that likely makes key it's looking for include the save name, so probably won't work.

I think the "correct" way would be to use a format string replacement instead of string concatination - IE something like "messageBoxContent = format(tr("Do you really want to override %s?"), metadataSaveName)" 

so the translation replacement string is "Do you really want to override %s?", and the translated string also has a replacement tag so still works as expected.

Reply:
Updated:

```c++
	const auto &metadataSaveName = saveMetadata.getName();
	const auto messageBoxTitle = tr("Override saved game");
	const auto messageBoxContent =
	    format(tr("Do you really want to override %s?"), metadataSaveName);
```